### ISSUE TITLE
Keyboard order doesn't match visual order for control panel buttons

### ISSUE THREAD

**Issue by zepumph** [phetsims/ohms-law#93]
@ariel-phet, as lead designer, I have a question.

Current button layout:
![image](https://user-images.githubusercontent.com/6856943/31407241-adefc2ee-adb0-11e7-9689-9db437407e90.png)

Current Keyboard nav order:
* control panel sliders
* sound toggle button
* reset all button

These don't align for visual keyboard users. Up until this point the protocol/convention is that the reset all button should be the last item in the keyboard order before the nav bar. I think this is a good convention, but then it seems we are a bit stuck, unless we switch the buttons.

Should we switch the buttons?

Tagging @terracoda @jessegreenberg @emily-phet in case someone holds strong opinions.

**Comment by ariel-phet** [comment]
@zepumph yes it would be fine to switch those. In fact, probably desirable since the reset all button generally is located in the lower right of the sim.

**Comment by terracoda** [comment]
@zepumph, glad you brought this up. I was wondering about it, too. In John Travoltage, the buttons are the reversed.

**Comment by zepumph** [comment]
Sounds great! I love these quick issues to tackle questions about the sim. Thanks for fast turnaround on feedback.
